packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/fixtures/UiTestExtensions.kt

```diff
@@ -65,6 +71,10 @@ private class UiTestExtension :
     ): Any = remoteRobot
 
     override fun beforeAll(context: ExtensionContext?) {
+        if (!StepWorker.processors.containsElements { it is StepLogger }) {
+            StepWorker.registerProcessor(StepLogger())
+        }
```

Comment:
What is a StepWorker? and what is this piece doing?

Reply:
Also is it somehow related to the step calls you are doing in [IdeaFrame.kt](https://github.com/mongodb-js/intellij/pull/56/files#diff-0f6b1cf984fc49ad87c70eec12f076ccc20b131be0c6e016278bc40bbeb28869)

Reply:
ah, this is for logs. This will show all the `steps` that we've done in the test in the logs.